docs/resources/destination_snowflake.md

```diff
@@ -0,0 +1,144 @@
+---
+page_title: "rudderstack_destination_snowflake Resource - terraform-provider-rudderstack"
+subcategory: ""
+description: |-
+  
+---
+
+# rudderstack_destination_snowflake (Resource)
+
+This resource represents a Snowflake destination. For more information check 
+https://www.rudderstack.com/docs/destinations/warehouse-destinations/snowflake/
+
+## Example Usage
+
+```terraform
+resource "rudderstack_destination_snowflake" example{
+  name = "my-snowflake"
+
+  config {
+    account = "..."
+    database = "..."
+    warehouse = "..."
+    user = "..."
+    password = "..."
+    sync {
+      frequency = "60"
+      # start_at                  = "10:00"
+      # exclude_window_start_time = "11:00"
+      # exclude_window_end_time   = "12:00"
+    }
+    # json_paths = "..."
+    use_rudder_storage = true
+    # namespace = "..."
+    # prefix = "..."
+    # additional_properties = true
+    # s3 {
+    #   bucket_name = "..."
+    #   access_key_id = "..."
+    #   access_key = "..."
+    #   enable_sse = true
+    # }
+    # gcp {
+    #   bucket_name = "..."
+    #   credentials = "..."
+    #   storage_integration = "..."
+    # }
+    # azure {
+    #   container_name = "..."
+    #   account_name = "..."
+    #   account_key = "..."
+    #   storage_integration = "..."
+    # }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `config` (Block List, Min: 1, Max: 1) Destination specific configuration. Check the nested block documenation for more information. (see [below for nested schema](#nestedblock--config))
+- `name` (String) Human readable name of the destination. The value has to be unique across all destinations.
+
+### Optional
+
+- `enabled` (Boolean) An enabled destination allows data to be sent to it.
+
+### Read-Only
+
+- `created_at` (String) Time when the resource was created, in ISO 8601 format.
+- `id` (String) The ID of this resource.
+- `updated_at` (String) Time when the resource was last updated, in ISO 8601 format.
+
+<a id="nestedblock--config"></a>
+### Nested Schema for `config`
+
+Required:
+
+- `account` (String) Account ID of your Snowflake warehouse. This account ID is part of the Snowflake URL.
```

Comment:
Can we add the FAQ describing the various Snowflake account ID examples here?

https://www.rudderstack.com/docs/destinations/warehouse-destinations/faq/#while-configuring-the-snowflake-destination-what-should-i-enter-in-the-account-field

Reply:
This is an auto generated doc generated from the go file. I couldn't find a way to add the example.

Reply:
I think that should be fine!

Reply:
done